lib/BackbeatConsumer.js

```diff
@@ -107,13 +118,17 @@ class BackbeatConsumer extends EventEmitter {
             circuitBreakerMetrics: joi.object({
                 type: joi.string().required(),
             }).optional(),
+            // if true, ensures that the processing of messages with the same key
+            // is done in order, messages with no key are processed in parallel
+            ordered: joi.boolean().default(true),
```

Comment:
i wonder if this should be a boolean, or if we should have a callback: to allow different keying strategies where needed (null --> no order, or callback to get the key, etc...)